presto-main/src/main/java/io/prestosql/util/MorePredicates.java

```diff
@@ -29,4 +29,9 @@ private MorePredicates() {}
         }
         return predicate;
     }
+
+    public static Predicate<Boolean> identity()
```

Comment:
This cannot be static imported together with `java.util.function.Function#identity`.

Reply:
can we rename the function to `identityPredicate` ?